### DISCUSSION TITLE
daisyUI: [postcss] Cannot find module '../dist/utilities'

### DISCUSSION THREAD

**Discussion by abhimaurya-dev** [saadeghi/daisyui#2544] [Q&A] [answered]
Getting Error as in title when adding daisyui to tailwind.config.js
I am using React with vite
Error:-
![image](https://github.com/saadeghi/daisyui/assets/100673182/d7ed7e2e-d952-4de7-aa3f-be6179e8b0f3)
My tailwind.config.js
![image](https://github.com/saadeghi/daisyui/assets/100673182/585b589b-32ff-449f-93a5-14556ed59ee9)

**Comment by saadeghi** [comment, marked as answer]
Thanks.  
It must be fixed now. update to the latest version
```
npm i -D daisyui@latest
```
